alacritty/src/migrate.rs

```diff
@@ -124,7 +124,23 @@ fn migrate_imports(
             Ok(import) => import,
             Err(err) => return Err(format!("import error: {err}")),
         };
+
+        // Keep yaml import if path does not exist.
+        if !import.exists() {
+            if options.dry_run {
+                eprintln!("Keeping yaml config for nonexistent import: {import:?}");
+            }
+            new_imports.push(Value::String(import.to_string_lossy().into()));
+            continue;
+        }
+
         let new_path = migrate_config(options, &import, recursion_limit - 1)?;
+
+        // Print new import path.
+        if options.dry_run {
+            println!("Successfully migrated import {import:?} to {new_path:?}");
+        }
```

Comment:
Just added this because it seems like a good idea to have.

Reply:
Do we need it only for `dry-run` though?

Reply:
Otherwise it's printed twice. Since every wet run includes a dry run, we only log stuff during the dry run.